samples/samples/admin/samples.py

```diff
@@ -61,16 +61,94 @@ def create_instance(instance_id):
 # [END spanner_create_instance]
 
 
+# [START spanner_create_instance_with_processing_units]
+def create_instance_with_processing_units(instance_id, processing_units):
+    """Creates an instance."""
+    spanner_client = spanner.Client()
+
+    config_name = "{}/instanceConfigs/regional-us-central1".format(
+        spanner_client.project_name
+    )
+
+    request = spanner_instance_admin.CreateInstanceRequest(
+        parent="projects/{}".format(spanner_client.project),
+        instance_id=instance_id,
+        instance=spanner_instance_admin.Instance(
+            config=config_name,
```

Comment:
`name` is not passed here, this could get passed as "" (empty) by default. Not sure if this will lead to some wierd result. can you please check.

Reply:
Name can be omitted here 
https://github.com/googleapis/googleapis/blob/master/google/spanner/admin/instance/v1/spanner_instance_admin.proto#L997-L998

Similar Go sample https://github.com/GoogleCloudPlatform/golang-samples/blob/main/spanner/spanner_snippets/spanner/spanner_create_instance.go#L37-L46